src/js/abstract/component-morph.js

```diff
@@ -0,0 +1,173 @@
+import morph from './morph';
```

Comment:
we will create it's own npm package for morphin in the future

Reply:
Maybe add a todo in the file directly and also WHY we did copy/modify the nanomorph one